lighthouse-core/fraggle-rock/config/filters.js

```diff
@@ -20,6 +20,60 @@ const baseArtifactKeySource = {
 
 const baseArtifactKeys = Object.keys(baseArtifactKeySource);
 
+// Some audits are used by the report for additional information.
+// Keep these audits unless they are *directly* skipped with `skipAudits`.
+const filterResistantAuditIds = ['full-page-screenshot'];
```

Comment:
One of the TODOs you removed mentioned `Stacks`, it's not a base artifact in FR. Should we include here?

Reply:
Ah, that TODO was actually solved by a previous update to filtering, but the TODO was missed in another file :)

This is a set of *audit* IDs that should be kept even if they're not in the `onlyAudits`/`onlyCategories` list. Extra artifacts are never kept unless they are base artifacts.